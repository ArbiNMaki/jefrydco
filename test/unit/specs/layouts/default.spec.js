import { shallowMount } from "@vue/test-utils";

import Default from "~/layouts/default";

describe("Default layout", () => {
  it("renders to match snapshot", () => {
    const wrapper = shallowMount(Default);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
