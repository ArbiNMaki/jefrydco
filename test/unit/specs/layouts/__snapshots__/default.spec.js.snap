// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default layout renders to match snapshot 1`] = `
<div class="container mx-auto">
  <app-cookie-stub></app-cookie-stub>
  <div></div>
</div>
`;
